site/content/docs/accounts-and-users/creating-api-key.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Creating an API key
+weight: 2
+menu:
+  docs:
+    parent: "Accounts & Users"
+aliases:
+- "/docs/accounts-users/creating-api-key/"
+---
+
+The Checkly Public API uses API keys to authenticate requests. API keys are unique to the user and not tied to accounts. 
+
+Be aware that Account-based API keys will be deprecated soon. Please create an API key for your user account in [User Settings](https://app.checklyhq.com/settings/user/).
```

Comment:
```suggestion
Be aware that account-based API keys will be deprecated soon. Please create an API key for your user account in [User Settings](https://app.checklyhq.com/settings/user/).
```

Reply:
@chocolateofpain let's call this out with a ">"